client/consensus/src/tests.rs

```diff
@@ -15,469 +15,44 @@
 // along with Tanssi.  If not, see <http://www.gnu.org/licenses/>.
 
 #![allow(clippy::await_holding_lock)]
```

Comment:
I know this was inherited from parity, but if we can get rid of it it would be great

```suggestion
```

Reply:
fixed in [82242eb](https://github.com/moondance-labs/tanssi/pull/662/commits/82242eb5310be2e51fad9bb311e231f63c8199bc)